### PR TITLE
Initial commit for the landing page feature

### PR DESCRIPTION
**Pull request by emeka-osuagwu**

### CODE REVIEW

File: app/Http/routes.php [hunk 1]
```diff
@@ -12,5 +12,5 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return view('app.feed');
```
Comment (unicodeveloper) [inline app/Http/routes.php:15]: Please change the name from feed to something more appropriate!

File: config/database.php [hunk 2]
```diff
@@ -68,10 +68,10 @@
...
         'pgsql' => [
             'driver'   => 'pgsql',
-            'host'     => env('DB_HOST', $url["host"]),
-            'database' => env('DB_DATABASE', substr($url["path"], 1)),
-            'username' => env('DB_USERNAME', $url["user"]),
-            'password' => env('DB_PASSWORD', $url["pass"]),
+            'host'     => env('DB_HOST', localhost),
+            'database' => env('DB_DATABASE', 'thehub'),
+            'username' => env('DB_USERNAME', 'postgres'),
+            'password' => env('DB_PASSWORD', 'Olanicmi2013'),
```
Comment (busayo) [inline config/database.php]: i need you to revert this commit. You are exposing your credentials to the public. To revert, you can check for more infomation on that [here](http://goodheads.io/2015/08/18/think-like-a-git-commit-like-a-boss-revert-now/)

File: public/custom/javascripts/init.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  /********************************************************************
+  / Function to show and hide top nav in mobile  view
+  */
+  $(function()
+  {
+    $('.button-collapse').sideNav();
+
+  });
+
+
```
Comment (unicodeveloper) [inline public/custom/javascripts/init.js:12]: Rename this folder to `javascript`. There is nothing like `javascripts`

File: resources/views/app/feed.blade.php [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+<div class="row">
+    @include('app.includes.overall.side_nav')
+    @include('app.includes.feeds.body')
+</div>
+
+@include('app.includes.overall.footer')
+
+
+
+
```
Comment (unicodeveloper) [inline resources/views/app/feed.blade.php:19]: Rename this file to `index.blade.php`, Use `master` in place of `overall` . Reduce the space from line 5 to line 10. I need you to restructure this using `@yield` and `@extends`

File: resources/views/app/includes/overall/top_nav.blade.php [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      <li><a href="sass.html"><i class="material-icons">add_alert</i></a></li>
+
+      <li>
+        <img style = "width:40px;margin-top:11px;"src="https://fbcdn-profile-a.akamaihd.net/hprofile-ak-xtp1/v/t1.0-1/p160x160/11816977_874992065923170_7087418417398847048_n.jpg?oh=27ba13e6365cd47f6ff964e91669a7d4&oe=569FF650&__gda__=1449787292_64e468d16774a38603c37ac775973e2c" alt="" class="circle responsive-img">
+      </li>
+
+      <!-- Dropdown Trigger -->
+      <li>
+        <a class="dropdown-button" href="#!" data-activates="dropdown1">Account
+            <i class="material-icons right view_headline">view_headline</i>
```
Comment (unicodeveloper) [inline resources/views/app/includes/overall/top_nav.blade.php:25]: Why does the nav menu have an underscore in its name?

File: server.sublime-workspace [hunk 6]
```diff
@@ -0,0 +1,329 @@
...
+	},
+	"show_minimap": false,
+	"show_open_files": false,
+	"show_tabs": true,
+	"side_bar_visible": true,
+	"side_bar_width": 268.0,
+	"status_bar_visible": true,
+	"template_settings":
+	{
+	}
```
Comment (unicodeveloper) [inline server.sublime-workspace:328]: Why did you commit this file?..You want to impose your workspace config on every one? :angry:  Please remove!

**Comment by unicodeveloper** [comment]
The title of your pull request and even your commits do not follow the conventions for this project. Please Fix!

**Comment by unicodeveloper** [comment]
Since there is no activity here..I'll close this PR!

**Pull request closed without merging**